File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        int temp = 0;
        boolean found = false;
        for (int i = 0; i < nums.length ; i++){
            temp = nums[i];
            for (int j = 0; j < nums.length; j++){
                if (i != j && temp == nums[j]){
                    found = true;
                    break;
                }
            }
            if (found) break;
        }
        return found;
    }

}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        count = Counter(nums)
        freq = [[] for _ in range (len(nums)+1)]
        for num,freq_count in count.items():
            freq[freq_count].append(num)
        res = []
        for i in range(len(freq)-1,0,-1):
            for num in freq[i]:
                res.append(num)
                if len(res) ==k:
                    return res
        return res
        
        